.github/workflows/release_tag.yml

```diff
@@ -0,0 +1,76 @@
+name: Release Tag
+
+on:
+  # Runs when a release is published
+  # Pushes a tagged image
+  # That is deployed to the "staging/production" environments
+  release:
+    types: [published]
+  # TODO REMOVE THIS BEFORE MERGING
+  pull_request:
+
+jobs:
+  context:
+    runs-on: ubuntu-latest
+
+    outputs:
+      is_fork: ${{ steps.context.outputs.is_fork }}
+
+    steps:
+      - uses: actions/checkout@v4
+      - id: context
+        uses: ./.github/actions/context
+
+  build:
+    needs: context
+    uses: ./.github/workflows/build.yml
+    with:
+      image: us-docker.pkg.dev/moz-fx-amo-prod/amo-prod/addons-server
+      target: production
+
+  test:
+    needs: build
+    uses: ./.github/workflows/test.yml
+    with:
+      artifact_name: ${{ needs.build.outputs.artifact_name }}
+
+  push:
+    if: ${{ needs.context.outputs.is_fork == 'false' }}
+    needs: [context, build, test]
+    runs-on: ubuntu-latest
+
+    # Add "id-token" with the intended permissions.
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - id: image
+        uses: ./.github/actions/download-docker
```

Comment:
I had a question in https://github.com/mozilla/addons-server/pull/22423, which got closed. But the question remains so I'll ask it here again. :)

> What I don't understand is why the image is pulled from Dockerhub and then pushed to GAR. Isn't this what the pull-verify-push script in the deploy-amo repo is already doing?

> I was expecting the workflow to be
> 1. build the image
> 2. push the image to both Dockerhub and GAR

Thanks.

Reply:
In this case it is not being pulled from dockerhub, but from a github artifact. The reason it is being pulled from a github artifact is to solve a totally different problem, which is how to use pushed images in our CI with forked PRs, which have no access to secrets and so cannot authenticate.

Reply:
To be clear, this is still WIP.